cluster/health-monitor.sh

```diff
@@ -27,13 +32,14 @@ CHECK_PERIOD=${CHECK_PERIOD:-10}
 # and containerd.
 SLEEP_SECONDS=${SLEEP_SECONDS:-120}
 
+sleep ${INITIAL_WAIT_SECONDS}
 while true; do
   # Use crictl pods because it requires both containerd and
   # cri-containerd to be working.
   if ! timeout ${COMMAND_TIMEOUT} ${CRICTL} pods > /dev/null; then
     echo "crictl pods timeout!"
-    pkill containerd
-    pkill cri-containerd
+    pkill -x containerd
```

Comment:
why kill containerd first?

Reply:
Ha, yeah, we may want to kill cri-containerd first. Make sense.

Reply:
Done.